StreamingAnalytics/ResourceMonitoring/resource-monitor.py

```diff
@@ -0,0 +1,30 @@
+import psutil
+import paho.mqtt.client as mqtt
+import time
+
+mqttClient = mqtt.Client("Resource-monitor")
+broker = "localhost"
+
+def on_publish(client,userdata,result):             #create function for callback
+	print("data published \n")
+	pass
+
+def on_connect(client, userdate, flags, rc):
+	print("Connected to MQTT broker on " + broker + " with result {0}".format(rc))
+
+mqttClient.on_connect = on_connect
+mqttClient.on_publish = on_publish
+mqttClient.connect(broker)
+
+while True:
+	cpuPercent = psutil.cpu_percent()
```

Comment:
I don't think we're using this correctly:
https://psutil.readthedocs.io/en/latest/#psutil.cpu_percent
"When interval is 0.0 or None compares system CPU times elapsed since last call or module import, returning immediately. That means the first time this is called it will return a meaningless 0.0 value which you are supposed to ignore."
So I would advise also calling psutil.cpu_percent() before the loop and putting the time.sleep(5) at the start of the loop.